extensions/wikia/PortableInfobox/controllers/ApiQueryAllinfoboxes.class.php

```diff
@@ -34,4 +35,19 @@ public function execute() {
 	public function getVersion() {
 		return __CLASS__ . '$Id$';
 	}
+
+	/**
+	 * @desc As a infobox template label we want to return a nice, clean text, without e.g. '_' signs
+	 * @param $text infobox template title
+	 * @return String
+	 */
+	private function createLabel( $text ) {
```

Comment:
Assuming we don't have code that already does this in the repo, LGTM. Did you check?